src/components/lightbox/lightbox.js

```diff
@@ -0,0 +1,65 @@
+import mixin from '../../globals/js/misc/mixin';
+import createComponent from '../../globals/js/mixins/create-component';
+import initComponentBySearch from '../../globals/js/mixins/init-component-by-search';
+import eventMatches from '../../globals/js/misc/event-matches';
+
+class Lightbox extends mixin(createComponent, initComponentBySearch) {
+  constructor(element, options) {
+    super(element, options);
+    this.activeIndex = this.element.dataset.lightboxIndex;
+    this.totalSlides =
+      [...this.element.querySelectorAll(this.options.selectorLightboxItem)].length - 1;
```

Comment:
`NodeList` (an array-like that qSA returns) has `.length` property, so you don’t need to convert it to an array.

Reply:
Thank you! Fixed